dsgrid/query/query_submitter.py

```diff
@@ -309,11 +314,17 @@ def submit(
             Raised if the model defines a project version
         DSGInvalidQuery
             Raised if the query is invalid
-
         """
-        return self._run_query(
-            model, load_cached_table, persist_intermediate_table, zip_file=zip_file, force=force
-        )[0]
+        scratch_dir = scratch_dir or DsgridRuntimeConfig.load().get_scratch_dir()
```

Comment:
Do we have documentation for where the user should specify their scratch directory?

Reply:
```
dsgrid —help
…
  -s, --scratch-dir TEXT    Base directory for dsgrid temporary directories.
                            Must be accessible on all compute nodes. Defaults
                            to the current directory.
```